src/service/services/user/user.service.ts

```diff
@@ -1,26 +1,23 @@
-import { User } from '../../entity/user.entity'
```

Comment:
i cannot understand why you created folder called service and locate your services inside one more services directory. I think services should be located in src folder. Middlewares also are usually in src. As for configurations files it is more common to create general folder for configs that will contain all configurations for the application but you created separated configs for data-access and service. These mistakes are not very serious, but they make it difficult to navigate through the project folder structure

Reply:
In this folders i was trying to distribute layers of app (to create architecture test like files in one folder don't depend on file in the other folder), like data-access, router (presentation layer) and service (business logic layer).

Reply:
your business logic should be located only in services folder (your schedule also includes business rules so it can be considered part of business logic) but you have added middlewares and config that are not related to business to 'service' folder, so i think your project structure is a bit incorrect